### ISSUE TITLE
Problem with DCOS marathon Virtual network

### ISSUE THREAD

**Issue by vimalprakashts** [mesosphere/marathon#4875]
hi all,

I installed DCOS with 3 masters and 3 agents.

I facing problem with virtual network . 

here is my json. generated by marathon in dcos. 

```
{
  "id": "/nginx",
  "cmd": null,
  "cpus": 1,
  "mem": 128,
  "disk": 0,
  "instances": 1,
  "container": {
    "type": "DOCKER",
    "volumes": [],
    "docker": {
      "image": "nginx",
      "network": "BRIDGE",
      "portMappings": [
        {
          "containerPort": 80,
          "hostPort": 0,
          "servicePort": 10002,
          "protocol": "tcp",
          "name": "main1",
          "labels": {
            "VIP_0": "9.0.0.0:34562"
          }
        }
      ],
      "privileged": false,
      "parameters": [],
      "forcePullImage": false
    }
  },
  "portDefinitions": [
    {
      "port": 10002,
      "protocol": "tcp",
      "labels": {}
    }
  ]
}
```
This I can see in dcos virtual network section.

VIRTUAL NETWORK NAME |	SUBNET	  |   AGENT PREFIX LENGTH
dcos	                                9.0.0.0/8	          24

this containers stays in waiting for long time. if I remove the port mapping section it runs successfully. 

Basically I need to know how to work with this new virtual network, and fix the service discovery and load balancing without using any extra stuff. 

Please give me some suggestion on this.

**Comment by meichstedt** [comment]
Note: This issue has been migrated to https://jira.mesosphere.com/browse/MARATHON-4261. For more information see https://groups.google.com/forum/#!topic/marathon-framework/khtvf-ifnp8.

**Comment by meichstedt** [comment]
Note: This issue has been migrated to https://jira.mesosphere.com/browse/MARATHON-4261. For more information see https://groups.google.com/forum/#!topic/marathon-framework/khtvf-ifnp8.
